pages/common/rclone.md

```diff
@@ -3,27 +3,27 @@
 > Copy, synchronize or move files and directories to and from many cloud services.
 > More information: <https://rclone.org>.
 
+- Launch an interactive menu to setup rclone:
+
+`rclone config`
+
 - List contents of a directory on an rclone remote:
 
 `rclone lsf {{remote_name}}:{{path/to/directory}}`
 
-- Copy file or directory from local source to remote destination:
+- Copy file or directory from the local machine to remote destination:
 
-`rclone copy {{path/to/source_file_or_directory}} {{remote_name}}:{{path/to/destination_directory}}`
+`rclone copy {{path/to/source_file_or_directory}} {{remote_name}}:{{path/to/directory}}`
 
-- Copy file or directory from remote source to local destination:
+- Copy files changed within the past 24 hours to a remote to the local machine, asking user to confirm each file:
 
-`rclone copy {{remote_name}}:{{path/to/source_file_or_directory}} {{path/to/destination_directory}}`
+`rclone copy --interactive --max-age 24h {{remote_name}}:{{path/to/directory}} {{path/to/local_directory}} `
 
-- Sync local source to remote destination, changing the destination only:
+- Mirror file or directory (unlike copy, sync removes files from remote if it does not exist on local):
```

Comment:
```suggestion
- Mirror file or directory (unlike copy, sync it removes files from remote if it does not exist locally):
```

Reply:
I believe the word "it" would make it ungrammatical in this case (also for consistency "Note:" should also be added)